container-engine-lib/lib/backend_impls/docker/docker_network_allocator/docker_network_allocator_test.go

```diff
@@ -25,31 +25,15 @@ func TestErrorOnNoFreeIps(t *testing.T) {
 	networks := parseNetworks(t, cidrs)
 	_, err := findRandomFreeNetwork(networks)
 	assert.Error(t, err)
-
 }
 
 func TestEntireNetworkingSpace(t *testing.T) {
 	takenNetworks := []*net.IPNet{}
-	allPossibleNetworks := []*net.IPNet{}
-	for thirdOctet := thirdOctetLowestPossibleValue; thirdOctet <= thirdOctetHighestPossibleValue; thirdOctet++ {
-		ipAddressString := fmt.Sprintf("%v.%v.%v.0", allowedNetworkFirstOctet, allowedNetworkSecondOctet, thirdOctet)
-		resultNetworkIp := net.ParseIP(ipAddressString)
-		resultNetwork := &net.IPNet{
-			IP:   resultNetworkIp,
-			Mask: networkCidrMask,
-		}
-		allPossibleNetworks = append(allPossibleNetworks, resultNetwork)
-	}
-	i := 0
-	for {
-		if i == thirdOctetHighestPossibleValue {
-			break
-		}
+	for i := 0; i < (1 << enclaveWidthBits); i++ {
 		freeIPAddress, err := findRandomFreeNetwork(takenNetworks)
```

Comment:
can we assert that 1024 ip addresses are available for the 64 different networks that are possible?

Reply:
Done!